Homework/Quang_Minh/Lesson1/SortArrayByParity.java

```diff
@@ -0,0 +1,28 @@
+class SortArrayByParity {
+    public int[] sortArrayByParity(int[] nums) {
+        int size = nums.length;
+        int position = 0;
+        for (int i = 0; i < size; i++) {
+            if (nums[i] %2 == 0) {
+                if (position != i) {
+                    nums[i] = nums[position] + nums[i];
```

Comment:
should create a swap func and reuse to avoid dup code.

Reply:
I've fixed as your cmt. please check and approve my PR.